src/components/Card/sass/Card.scss

```diff
@@ -0,0 +1,118 @@
+@use 'sass:map';
+.tyk-card { 
+  border: 1px solid theme-color('background', 'base');
+  border-radius: 8px;
+  box-shadow: 2px 2px 3px theme-color('background', 'base');
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+
+  &__header {
+    align-items: center;
+    display: flex;
+    font-family: $font-inter-regular;
+    font-size: $sm-font-size;
+    padding: map.get($spacing, 'base') map.get($spacing, 'base') 0 map.get($spacing, 'base');
+  
+    &-right {
+      margin-left: auto;
+    }
+  }
+
+  &__title-group {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    align-items: center;
+
+    .tyk-card__title-icon {
+      align-items: center; 
+      display: flex;
+      height: 42px;
+      justify-content: center;
+      margin-right: 10px;
+      grid-row: span 2;
+      width: 42px;
+    }
+  }
+
+  &__title { 
+    font-family: $font-inter-bold;
+    font-size: $lg-font-size;
+    display: box;
```

Comment:
I had to look for the `box` value but it doesn't seem to exist. I think it should be `display: flex;` instead.

Reply:
https://css-tricks.com/almanac/properties/l/line-clamp/